src/private/recurseClassificationNode.ps1

```diff
@@ -11,21 +11,21 @@ function recurseClassificationNode {
 
         foreach ($node in $ClassificationNodes) {
             if ($ParentNodeSecurityToken) {
-                $token = "$($ParentNodeSecurityToken):vstfs:///Classification/Node/$($node.identifier)"
+                $token = "$($ParentNodeSecurityToken)::vstfs:///Classification/Node/$($node.identifier)"
```

Comment:
Single colon worked on up-level Azure DevOps versions. Let's add a version check to use double colons in security token.

Reply:
Tested on 2018